tests/test_decoder.py

```
@@ -524,333 +490,313 @@ def test_relation_format(self, relex_config, relex_inputs):
         # Masked positions should not produce relations
         # In batch 0, positions 2 and 3 are masked out
         # This is indirect - we just verify no errors occur
         assert isinstance(relations, list)
-    
+
     def test_no_relations_when_not_requested(self, relex_config, relex_inputs):
         """Should return empty relations when rel_id_to_classes is None."""
         decoder = SpanRelexDecoder(relex_config)
-        
+
         spans, relations = decoder.decode(
```

Ruff (0.15.1)

[warning] Unpacked variable `spans` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)